### ISSUE TITLE
Potential false negative for extra semi-colon error

### ISSUE THREAD

**Issue by GoogleCodeExporter** [google/closure-linter#15]
```
if (true)
 alert('yes');
else;
 alert('no');

Does not report that else shouldn't be followed immediately by a semi-colon. 
This might be technically acceptable JavaScript (doesn't throw an error) but is 
probably never intended.


```

Original issue reported on code.google.com by `a...@google.com` on 8 Nov 2010 at 5:23
- Merged into: #14
